schema/data/dev/certification_url.sql

```diff
@@ -23,11 +26,16 @@ update ggircs_portal.certification_url set certification_signature = 'signed' wh
 
 alter table ggircs_portal.application_revision_status disable trigger _status_change_email;
 alter table ggircs_portal.application_revision_status disable trigger _read_only_status_for_non_current_version;
+
 -- Update the status of application with id=1 to be 'submitted'
-  insert into ggircs_portal.application_revision_status(application_id, version_number, application_revision_status)
-    values (1,1,'submitted');
-  insert into ggircs_portal.application_revision_status(application_id, version_number, application_revision_status)
-  values (1,2,'submitted');
+insert into ggircs_portal.application_revision_status(application_id, version_number, application_revision_status)
+  values (1,1,'submitted');
+
+-- Moving timestamp one second forward to avoid revisions being submitted at the same time
+select mocks.set_mocked_time_in_transaction('2020-07-03 00:00:00.000000-07'::timestamptz + interval '1 second');
```

Comment:
how about
```suggestion
select mocks.set_mocked_time_in_transaction(now() + interval '1 second');
```

Reply:
Some tests rely on this data, so having a dependency on `now()` is not the best. We can have it like the previous comment, where we fetch the value from the `reporting_year` table

Reply:
Sounds good